src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -984,7 +984,18 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 								// Otherwise fall back to the bitfield requirements
 								RequireDynamicallyAccessedMembers (ref reflectionContext, requiredMemberTypes, value, calledMethodDefinition);
 							}
+
+							if (value is not LeafValueWithDynamicallyAccessedMemberNode { DynamicallyAccessedMemberTypes: DynamicallyAccessedMemberTypes.All })
```

Comment:
Mono run uses an analyzer that complains about the way this statement is written, so I will change it a bit